src/containers/buttons/GenerateSuiteButton.js

```diff
@@ -0,0 +1,143 @@
+import React, {Component} from "react";
+import {bindActionCreators} from "redux";
+import {connect} from "react-redux";
+import xmlbuilder from "xmlbuilder";
+import {saveAs} from "file-saver";
+
+
+class GenerateSuiteButton extends Component {
+
+  handleSuiteGenerating() {
+    const projectTests = this.props.project[0].tests;
+    const suiteElement = xmlbuilder.create('suite', {encoding: "utf-8"});
+    suiteElement.att('name', this.props.project[0].suite);
+    suiteElement.att('company', this.props.project[0].company);
+    suiteElement.att('domain', this.props.project[0].domain);
+    suiteElement.att('project', this.props.project[0].project);
+    if(projectTests) {
```

Comment:
You can merge this two conditionals to one

Reply:
Done.